<launch>
    <!--    <node pkg="tf" type="static_transform_publisher" name="world2odom"-->
    <!--          args="0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 1.0000 world odom 100"/>-->
    <!--    <node pkg="tf" type="static_transform_publisher" name="orbodom2odom"-->
    <!--          args="0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 1.0000 orb_odom odom 100"/>-->
    <node pkg="ORB_DVL2" type="twist2odom.py" name="twist2odom" />
    <node pkg="ORB_DVL2" type="stereo_dvl2" name="stereo_dvl2"
          args="$(find ORB_DVL2)/Vocabulary/ORBvoc.txt $(find ORB_DVL2)/data/underwater_orbslam3_rexrov.yaml false" output="screen"/>

    <!--    <node pkg="sonar_localization" type="sonar_black_node.py" name="ekf_se" clear_params="true" output="screen"/>-->
<!--    <arg name="model" default="$(find ORB_SLAM3_edit)/urdf/bluerov.urdf"/>-->
<!--    <param name="robot_description" command="$(find xacro)/xacro $(arg model)"/>-->
    <node name="rviz_dvl2" pkg="rviz" type="rviz" args="-d $(find ORB_DVL2)/launch/falcon_tightly.rviz"/>

</launch>